doc/sphinx/src/getting-started.rst

```diff
@@ -11,12 +11,13 @@ At it's most basic, you can download and compile ``singularity-eos`` with:
   cd singularity-eos
   mkdir bin
   cd bin
-  cmake ..
+  cmake -DSINGULARITY_FORCE_SUBMODULEMODE=ON -DSINGULARITY_USE_FORTRAN=OFF ..
   make -j 
   make install # optional: install into directory defined via CMAKE_INSTALL_PREFIX
 
 This will downloady singularity-eos with no optional dependencies and
-compile the capabilities available in that form.
+compile the capabilities available in that form. For more details, see
+our build page.
```

Comment:
Oh I see it now, could we provide a link to that page?

Reply:
I'll add one